cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
are these comments still useful? which information do we want to keep? (also see other files)

Reply:
I imagine we can just get rid of the example files altogether, or do you see any value in keeping them?

Reply:
yeah. I like less files and less to read better. thanks for going a step further